exampleSite/config.toml

```diff
@@ -12,7 +12,7 @@ googleAnalytics = ""
 [params]
   name = "Okkur Labs"
   description = "Open Source Theme for your next project"
-  attribution = "true"
+  attribution = "false"
```

Comment:
Do we want to make attribution opt-in? Reasoning behind this change?

Reply:
I could switch that back to true but add a comment to either enable true or false.

Reply:
Think it might be best to discuss that. Let's leave it as opt-in for now. Discussion is at #53